Tools/vagrant/initvagrant.sh

```diff
@@ -82,5 +82,8 @@ cd jsbsim
 make -j2
 EOF
 
+#Plant a marker for sim_vehicle that we're inside a vagrant box
+touch /ardupilot.vagrant
```

Comment:
I believe the EOF is an indicator that the virtual "file" sent to the sh command has ended.  It's not the EOF of the bash file in which this is written in